graph.js

```diff
@@ -76,8 +92,7 @@ class Graph {
                         return;
                     }
                     const pos = d3.pointer(event, graph.plot.node())
-                    const node = { id: ++this.nodeId, nodeType: "entity", "type": entityType, title: "", x: pos[0], y: pos[1], attributes: [{"key": "123", "value": "123", "type": "string"}]}
-                    node.title = node.type + "-" + node.id
+                    const node = new GraphNode(++this.nodeId, "entity", entityType, [], pos)
```

Comment:
I think we can leave the test attributes there for now? They are helpful for dev.

Reply:
sure thang